.circleci/scripts/release-create-gh-release

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+if [[ "${CI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CI environment variable must be set to true'
+    exit 1
+fi
+
+if [[ "${CIRCLECI:-}" != 'true' ]]
+then
+    printf '%s\n' 'CIRCLECI environment variable must be set to true'
+    exit 1
+fi
+
+printf '%s\n' 'Installing hub CLI'
```

Comment:
This step should probably be inside the `if` block below, for when a release commit is detected. Otherwise this would run on each commit to the `develop` branch, wouldn't it?

Reply:
Yes, good point